.github/workflows/php-hook-documentation.yml

```diff
@@ -0,0 +1,57 @@
+name: PHP Hook Documentation Generator
+
+on:
```

Comment:
The push and pull_request (with the PHP filter), look fine to me. But I was wondering if it would do any harm to also allow us to run this manually, like [here](https://github.com/woocommerce/google-listings-and-ads/blob/2.5.2/.github/workflows/e2e-tests.yml#L10). It's not going to effect our automated flows, and will make it easier if we ever want to run it manually.

Reply:
I implemented this suggestions